File: XGunSonraHaftaninHangiGunu/XGunSonraHaftaninHangiGunu.java
package xgunsonrahaftaninhangigunu;

import java.util.Scanner;

public class XGunSonraHaftaninHangiGunu {

    public static void main(String[] args) {
        String[] gunler = {"Pazartesi", "Sali", "Carsamba", "Persembe", "Cuma", "Cumartesi", "Pazar"};
        Scanner reader = new Scanner(System.in);
        System.out.print("Bu günün ne olduğunu girin: ");
        String bugun = reader.nextLine();
        System.out.print("Kaç gün sonrasını istediğinizi girin: ");
        int kacGun = reader.nextInt();
        int temp = 0;
        for (int i = 0; i < gunler.length; i++) {
            if (bugun.equalsIgnoreCase(gunler[i])) {
                temp = i;
            }
        }
        temp = (temp + kacGun) % 7;
        System.out.println(bugun + " gününden " + kacGun + " gün sonraki gün :  " + gunler[temp]);
    }
}

File: XGunSonraHaftaninHangiGunu/XGunSonraHaftaninHangiGunu.py
def hangi_gun(gun, ilerleme_sayisi):

    gunler = ["pazartesi", "salı", "çarşamba", "perşembe", "cuma", "cumartesi", "pazar"]
    gun = gun.lower()

    if gunler.__contains__(gun):
        gun_sayisi = (gunler.index(gun)+ilerleme_sayisi) % 7
        print("{} gününden {} gün sonra haftanın {}. günü olan {} günüdür.".format(gun.upper(), ilerleme_sayisi, gun_sayisi+1, gunler[gun_sayisi].upper()))
    else:
        print("Böyle bir gün bulunmamakta")


hangi_gun(input("Hangi günde olduğunuzu girin : "), int(input("Kaç gün sonrası olduğunu girin : ")))